Radiant/fileWatcher.js

```diff
@@ -216,9 +218,17 @@ const uploadFile = function (info, endStream){
  * @returns {PromiseLike<T | T | never> | Promise<T | T | never>}
  */
 const makeCopy = function(source, destination) {
-    return fs.copyFile(source, destination).then(() => {
-        return fs.appendFile(destination, '#EXT-X-ENDLIST\n').then(() => {
-            return destination;
+    return new Promise((resolve, reject) => {
+        fs.copyFile(source, destination).then(() => {
+            // insert #EXT-X-START:TIME-OFFSET=0 at line 5 so we start at beginning and not from 'live'
+            // or most recent segment
+            insertLine(destination).content('#EXT-X-START:TIME-OFFSET=0').at(5).then((err) => {
+                if(err) {
+                    Logger.error(err);
+                    reject(err);
```

Comment:
probably want to return here so you don't also call resolve

Reply:
reject kicks out of the promise.. pretty sure it wont call resolve.

Reply:
well found this.. lol so i guess its just good practice.. even though it wont `resolve` if the `reject` is called. 

https://stackoverflow.com/questions/32536049/do-i-need-to-return-after-early-resolve-reject